catalyst/dl/experiments/experiment.py

```diff
@@ -70,13 +69,18 @@ def get_model_stuff(self, model, stage: str):
     def get_callbacks(self, stage: str) -> "List[Callback]":
         pass
 
+    @abstractmethod
```

Comment:
all we need during training is loaders, so I am not sure it is good idea to force user to implement `get_datasets` and `get_datasets_only`

Reply:
`get_datasets_only` is implemented in `ConfigExperiment` and users should not reimplement it in most cases.  
May be we should update our config structure to separrate num workers, batch size etc from the rest of data params to make get_loaders and get_datasets independent?